docs/azure/sdk/logging.md

```diff
@@ -218,18 +218,24 @@ When troubleshooting unexpected behavior with a client library, it's helpful to
 - The HTTP request body sent to the underlying Azure service's REST API.
 - The HTTP response body received from the Azure service's REST API.
 
-By default, logging of the aforementioned content is disabled. To enable logging of the HTTP request and response bodies, complete the following steps:
+By default, logging of the aforementioned content is disabled. Libraries for Azure services like Event Hubs, Service Bus, and Web PubSub don't support this type of logging. Those libraries are based on alternative protocols, such as AMQP.
```

Comment:
It feels like this new sentence should be added higher up where we discuss HTTP, and added as some sort of note. 

Ex:
(Note: These options only apply to client libraries utilizing HTTP to communicate with the service. Libraries based on alternative protocols, such as AMQP, like Event Hubs, Service Bus, and Web PubSub don't support this type of logging.
```